test/api.js

```diff
@@ -1839,4 +1839,139 @@ describe('api', function() {
       done();
     });
   });
+
+  describe('binding', function() {
+    beforeEach(function() {
+      delete require.cache[sassPath];
```

Comment:
I think @xzyfer removed this stuff in a recent PR, so I'll let him go through this again